src/features/PersonCredits/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from "react";
+import { getPersonMovieCredits } from "../useApiKey"; // Import funkcji API
+import styled from "styled-components";
+const SectionTitle = styled.h2`
+  font-size: 24px;
+  margin: 20px 0 10px;
+`;
+const ListWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 15px;
+`;
+const ListItem = styled.div`
+  width: 150px;
+  text-align: center;
+`;
+const Image = styled.img`
+  width: 100%;
+  border-radius: 8px;
+`;
+export const PersonCredits = ({ personId }) => {
+  const [credits, setCredits] = useState({ cast: [], crew: [] });
+  const [error, setError] = useState(false);
+  useEffect(() => {
+    const fetchCredits = async () => {
+      const fetchedData = await getPersonMovieCredits(personId);
+      if (fetchedData) {
+        setCredits(fetchedData);
+      } else {
+        setError(true);
+      }
+    };
+    fetchCredits();
+  }, [personId]);
```

Comment:
Is it normal to mix these two (i mean API and styled-components) like that? Or mixing them at all?

Reply:
I will divide them into  separated files